modules/gutenberg/module.php

```diff
@@ -132,8 +133,49 @@ public function print_admin_js_template() {
 	 * @access public
 	 */
 	public function __construct() {
+		$this->register_experiments();
+
 		add_action( 'rest_api_init', [ $this, 'register_elementor_rest_field' ] );
 		add_action( 'enqueue_block_editor_assets', [ $this, 'enqueue_assets' ] );
 		add_action( 'admin_footer', [ $this, 'print_admin_js_template' ] );
+
+		add_action( 'wp_enqueue_scripts', [ $this, 'dequeue_assets' ], 999 );
+	}
+
+	public function register_experiments() {
+		Plugin::$instance->experiments->add_feature( [
+			'name' => 'gutenberg_assets_optimize',
+			'title' => esc_html__( 'Gutenberg Assets Optimize', 'elementor' ),
+			'description' => esc_html__( 'Optimize Gutenberg assets', 'elementor' ),
```

Comment:
Use this experiment to disable redundant file loading to optimize your site by excluding block editor files (styles and scripts) when they're not in use.